### DISCUSSION TITLE
Filtering by type of content?

### DISCUSSION THREAD

**Discussion by zsombro** [dailydotdev/daily#1131] [Feature Requests]
I like the idea that I can filter by topic, but it would be very helpful if I could filter out certain types of content as well: I'm seeing way too many beginner level tutorials and memes compared to the news pieces, articles and essays I'd like to primarily see. 

Other than that, I really like the extension!

**Comment by idoshamun** [comment]
Hi @zsombro,

Separating beginner and experienced developers content is a real challenge for us. We have some ideas on how to tackle that. As for memes, we know the platform has recently got a lot more memes which some devs really dig and others not so much (to say the least). We're trying to provide better filtering options. Stay tuned.
